--- ../../../../../app/views/repositories/_form.html.erb	2017-11-17 15:35:59.618339784 +0300
+++ _form.html.erb	2017-11-23 15:52:58.385611139 +0300
@@ -25,7 +25,7 @@
 <% end %>
 </div>
 
-<p>
-  <%= submit_tag(@repository.new_record? ? l(:button_create) : l(:button_save), :disabled => button_disabled) %>
+<p><strong><font color='red'><%= l(:text_button_does_not_create_repository) %></font></strong><br/>
+  <%= submit_tag(@repository.new_record? ? l(:button_add) : l(:button_save), :disabled => button_disabled) %>
   <%= link_to l(:button_cancel), settings_project_path(@project, :tab => 'repositories') %>
 </p>
